Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -359,7 +359,8 @@ void TBWidget::AddChildRelative(TBWidget *child, WIDGET_Z_REL z, TBWidget *refer
 
 void TBWidget::RemoveChild(TBWidget *child, WIDGET_INVOKE_INFO info)
 {
-    assert(child->m_parent);
+    if (!child->m_parent)
+        return;
 
```

Comment:
Ok, if we do keep the guard in, we should check that the parent == this, otherwise things can go sideways later in the call.  I am also seeing a crash when clicking the preview button on macOS, can provide a callstack tomorrow.